### PR TITLE
Sending correct Json Objects

### PR DESCRIPTION
**Pull request by AKidd95**
#### Description
Changes proposed in this pull request:

* Create Location Model
* Create associations between Location and user
* Add gender and age attributes to User table
* Validate and test attributes
* Format serializer to send correct JSON object data

#### What I have learned working on this feature: 
* The function of Serializers 
* how to structure JSON objects
* Associations are not so bad

### CODE REVIEW

**Review by NellaMori — COMMENTED** [review]
LGTM! 👍

File: app/controllers/api/v1/user_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Api::V1::UserController < ApiController
-  before_action :authenticate_api_v1_user!
+  # before_action :authenticate_api_v1_user!
```
Comment (magnus-thor) [inline app/controllers/api/v1/user_controller.rb]: uncomment this

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: spec/requests/api/v1/user_request_spec.rb [hunk 2]
```diff
@@ -20,9 +26,16 @@
...
-            create(:user)
+            user = create(:user)
+            user.languages.create(name: 'Dutch', native: false, learn: true)
+            location = Location.new(locale: 'France')
+            user.location = location
+            user.save
           end
         end
+
+        let(:users) { User.all }
```
Comment (tochman) [inline spec/requests/api/v1/user_request_spec.rb:37]:
```
 let(:users) do 
    2.times do
            user = create(:user)
            user.languages.create(name: 'Dutch', native: false, learn: true)
            location = create(:location, locale: 'France')
            user.location = location
            user.save
    end
end
```

File: app/controllers/api/v1/user_controller.rb (hunk 1 above)
Comment (AKidd95) [reply app/controllers/api/v1/user_controller.rb]: oh yeah thanks you

**Review by magnus-thor — APPROVED** [review]

**Pull request merged**